docs/continuous_integration.rst

```diff
@@ -0,0 +1,60 @@
+.. _ci_tests:
+
+Continuous Integration
+===============================
+
+The SecureDrop project uses `GitHub Actions <https://docs.github.com/en/actions>`_ for
+running automated continuous integration on code changes. You can get an overview of what
+each project does by reviewing the ``Makefile`` and files in the ``.github/workflows`` folder.
```

Comment:
I initially was a bit confused by which makefile and workflows folder this was referring to. I think the below suggestion is accurate, but let me know if I overlooked something.

```suggestion
each project does by reviewing the ``Makefile`` and files in the ``.github/workflows`` folder of the project's repository.
```